### PR TITLE
new triggered_after_koji_task parameter

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

**Comment by MartinBasti** [comment]
Please add ticket number and more details about this feature into commit message. Otherwise LGTM

File: osbs/build/build_requestv2.py [hunk 1]
```diff
@@ -105,6 +106,7 @@ def set_params(self, **kwargs):
         :param worker_deadline: int, worker completion deadline in hours
         :param orchestrator_deadline: int, orchestrator deadline in hours
         :param skip_build: bool, if we should skip build and just set buildconfig for autorebuilds
+        :param triggered_after_koji_task: str, koji task ID from which was autorebuild triggered
```
Comment (twaugh) [inline osbs/build/build_requestv2.py]: Wouldn't it be an integer?

Comment (rcerven) [reply osbs/build/build_requestv2.py]: will change

**Comment by rcerven** [comment]
updated commit message

File: osbs/api.py [hunk 2]
```diff
@@ -505,6 +505,9 @@ def _create_build_config_and_build(self, build_request):
                     imstreamtag['image']['dockerImageReference']
                 is_autorebuild = True
 
+            if build_request.triggered_after_koji_task is not None:
+                is_autorebuild = True
```
[the remark this answers is not in the document]
Comment (rcerven) [reply osbs/api.py]: we are passing triggered_after_koji_task, so we can use it later in koji build metadata, that way we will know from koji build which is the original task we are rebuilding

**Comment by rcerven** [comment]
rn updated: - New triggered_after_koji_task parameter, when provided set is_autorebuild in build config

**Pull request merged**